camel/personas/persona_hub.py

```diff
@@ -215,10 +216,29 @@ def is_similar(
         self, persona1: Persona, persona2: Persona, threshold: float
     ) -> bool:
         r"""Check if two personas are similar."""
-        # This is a placeholder. You should implement a proper similarity
-        # check, possibly using embedding-based methods as suggested in the
-        # paper.
-        return False  # Placeholder return
+        sentence_encoder = SentenceTransformerEncoder(
+            model_name='all-mpnet-base-v2'
+        )  # Using all-mpnet-base-v2, may switch to other models
```

Comment:
I suggest we support user define the embedding model they want and use openai embedding as default model since sentencetransformer would be heavy